tcf_website/views/team_info.json

```diff
@@ -113,6 +113,13 @@
       "class": "2023",
       "img_filename": "ENG_Nikhil_Mahajan.jpg",
       "github": "n4mahajan"
+    },
+    {
+      "name": "Borja Ingle-Fernandez",
+      "role": "Developer",
+      "class": "2023",
+      "img_filename": "ENG_Borja_Ingle-Fernandez.JPG",
```

Comment:
Not terribly important, but for the sake of consistency, this image file seems to be the only one with extension in uppercase.

Reply:
Should be fixed now.